File: Task6/ex2.py

using namespace std;

int main() {
    int a, d, e, f, des, b, c;
    
    cin >> a >> d >> e >> f;
    
   if (f + d < 10) {b = f + d; des = 0;} else {b = f + d - 10; des = 1;}
   if (des == 0) c = a - e; else c = a - e - 1;
   cout << b << " " << c;
}

File: 7class/Tema10/ex2.cpp
using namespace std;

int main() {

  int n, i, k;
  cin >> n;
  k=0;
  for (i=2; i<n; i++)
   {
      if (n%i==0) k++;
   }
  if (k==0 && n!=1) cout << "Yes"; else cout << "No";
}